libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/hwdef.dat

```diff
@@ -0,0 +1,176 @@
+# hw definition file for processing by chibios_hwdef.py
+#following exapmle: 
+#https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/mRoControlZeroH7/hwdef.dat#L105
+# for H743 bootloader
+
+AUTOBUILD_TARGETS None
```

Comment:
delete this line or it wont be autobuild on the the firmware server for all vehicle types